recipes/plumpy/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "plumpy" %}
+{% set version = "0.7.9"%}
```

Comment:
Looking on the Github page, there are releases for the newer versions that could be used:
https://github.com/aiidateam/plumpy/releases